atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2037,12 +2035,8 @@ bool WebContents::ExecuteScriptInTab(mate::Arguments* args) {
 
 bool WebContents::SendIPCSharedMemory(const base::string16& channel,
                                       base::SharedMemory* shared_memory) {
-  auto process_handle = web_contents()->GetRenderProcessHost()->GetHandle();
-  if (!process_handle)
-    return false;
-
-  base::SharedMemoryHandle memory_handle;
-  if (!shared_memory->ShareToProcess(process_handle, &memory_handle))
+  base::SharedMemoryHandle memory_handle = shared_memory->handle().Duplicate();
```

Comment:
Since https://chromium.googlesource.com/chromium/src/+/37e7d8301a078bd1b6b3ef7cffdb6c6ac3277edd, ShareToProcess is not needed anymore. Apparently, just duplicating the handle is enough.